lessons/pydata/pandas_types/index.ipynb

```diff
@@ -0,0 +1,10723 @@
+{
```

Comment:
<p>možná bych tu podotknul, že to není jen přes astype, ale že se to dá specifikovat už při nahrávání dat (a ušetřit si tim dost paměti už na začátku)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/36/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='73'/>

Reply:
<p>Můžu to vynechat?</p>

Reply:
Jj, je to celkem nedůležité a asi ze začátku na tyhle věci vůbec nenarazí.